apps/frontend/src/components/searchBar.tsx

```diff
@@ -122,6 +132,15 @@ const SearchBar = () => {
       page: 1,
       limit: 8,
     };
+    //NEED TO ADD THE TOKEN TO THE REQUEST
```

Comment:
the token is in the request by default as it is a cookie in the browser (if you're logged in)

Reply:
removed the comment